e2e/launch-simple-sns.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+#
+# Tested with dfx 0.20.1
+
+set -euo pipefail
+
+# This ID corresponds to TEST_NEURON_1
+NEURON_ID="449479075714955186"
+
+dfx start --clean --background
+
+dfx identity use default
+cargo run --bin nns install --dfx-cache-path="$(dfx cache show)"
+
+# Ensure we have a powerful neuron
+cat <<EOF >ident-1.pem
+-----BEGIN EC PRIVATE KEY-----
+MHQCAQEEICJxApEbuZznKFpV+VKACRK30i6+7u5Z13/DOl18cIC+oAcGBSuBBAAK
+oUQDQgAEPas6Iag4TUx+Uop+3NhE6s3FlayFtbwdhRVjvOar0kPTfE/N8N6btRnd
+74ly5xXEBNSXiENyxhEuzOZrIWMCNQ==
+-----END EC PRIVATE KEY-----
+EOF
+dfx identity import --force --storage-mode=plaintext ident-1 ident-1.pem
+dfx identity use ident-1
+PRINCIPAL_ID=$(dfx identity get-principal)
+
+# Hack
+sleep 15s
+
+# Top-up the SNS-W canister
+dfx ledger fabricate-cycles --canister qaa6y-5yaaa-aaaaa-aaafa-cai --t 2345
+
+dfx canister call "rrkah-fqaaa-aaaaa-aaaaq-cai" update_neuron '(
+  record {
+    id = opt record { id = '${NEURON_ID}' : nat64 };
```

Comment:
I was super confused about why the variable expansion works with the single quotes. But then I realized you actually end the single quote string, concatenate the variable and then open a new string. Is that intentional or did it work by accident?

Reply:
This was intentional, but I'm no Bash expert — maybe there's a better way.